infinitory/errors.py

```diff
@@ -0,0 +1,137 @@
+import os
+import pickle
+import sys
+import time
+
+from datetime import datetime
+
+import logging
+
+
+class ErrorParser(object):
+    def __init__(self):
+        self._reports = dict()
+        self._all_errors = []
+        self._unique_errors = []
+
+        self._logger = logging.getLogger()
+
+    def delete_report_cache(self):
+        cache_path = './cache/'
+        for f in os.listdir(cache_path):
+            # Delete cache item if older than 2 hours
+            if os.stat(os.path.join(cache_path,f)).st_mtime < time.mktime(datetime.now().timetuple()) - (1 * 3600):
+                print("Deleting File " + os.path.join(cache_path,f))
+                os.remove(os.path.join(cache_path,f))
+
+    def load_reports_fast(self, pupdb):
```

Comment:
Delete this function please.

Reply:
You could add a comment to the `load_reports` function explaining why you didn't use a subquery.

Reply:
Will do